lib/cartopy/crs.py

```diff
@@ -959,12 +960,30 @@ def __init__(self, central_longitude=0.0,
             If omitted, a default globe is created.
         latitude_true_scale: optional
             The latitude where the scale is 1. Defaults to 0 degrees.
+        false_easting: optional
+            X offset from the planar origin in metres. Defaults to 0.
+        false_northing: optional
+            Y offset from the planar origin in metres. Defaults to 0.
+        scale_factor: optional
+            Scale factor at natural origin. Defaults to unused.
 
+        Only one of ``latitude_true_scale`` and ``scale_factor`` should
+        be included.
         """
         proj4_params = [('proj', 'merc'),
                         ('lon_0', central_longitude),
-                        ('lat_ts', latitude_true_scale),
-                        ('units', 'm')]
+                        ('x_0', false_easting),
+                        ('y_0', false_northing),
+                        ('units', 'm'),
+                        ('lat_ts', latitude_true_scale)]
+        if scale_factor is not None:
+            if latitude_true_scale:
```

Comment:
You'll have to make the default `None` now; you can't assume the user didn't pass `scale_factor` and `latitude_true_scale=0` explicitly expecting it to work.

Reply:
*sigh* Yeah...